doc/api/esm.md

```diff
@@ -33,15 +33,14 @@ node --experimental-modules my-app.mjs
 ### Supported
 
 Only the CLI argument for the main entry point to the program can be an entry
-point into an ESM graph. In the future `import()` can be used to create entry
-points into ESM graphs at run time.
+point into an ESM graph. Dynamic import can also be used to create entry points
+into ESM graphs at run time.
```

Comment:
Only with an additional CLI flag `--harmony_dynamic_import`.

Reply:
like i said in the first comment, this might need to wait until that flag goes away

Reply:
I'd go for documenting it as it is to get this in right now.